### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Antoine Leresche and Robin Forestier

### DISCUSSION THREAD

**Discussion by A2va** [heig-vd-dai-course/heig-vd-dai-course#397] [Show and tell]
A command-line interface (CLI) tool that allows users to generate QR codes by providing text or URLs as input.
Features:
* Create QR codes from text / url / file / ...
* Display the QR in the terminal (ascii art)
* Save in a file (ascii art)
* Save as image
* Change size / shape / encoding

Practical work repository is available at https://github.com/A2va/dai-qr-code
Considered commit for evaluation: https://github.com/A2va/dai-qr-code/commit/7c9b6f3efb279644d47646364b7c9393f77ffc07

**Comment by ludelafo** [comment]
All good for me! What do you think @hadrylouis?

**Reply by hadrylouis** [reply]
All good for me too :)
